.github/workflows/ci.yaml

```diff
@@ -136,13 +136,13 @@ jobs:
               -o -iname '*.cl' \
               -o -iname '*.c' \
           | grep -v 'build/' \
-          | xargs -n 1 -P 1 -I{} -t sh -c 'clang-format-10 -style=file {} | diff - {}'
+          | xargs -n 1 -P 1 -I{} -t sh -c 'clang-format -style=file {} | diff - {}'
```

Comment:
We should use the executable with the version in the name. Also, if we are changing versions then the githook needs to be updated.